helix-term/src/commands.rs

```diff
@@ -2085,269 +2085,267 @@ mod cmd {
     pub const TYPABLE_COMMAND_LIST: &[TypableCommand] = &[
         TypableCommand {
             name: "quit",
-            alias: Some("q"),
+            aliases: &["q"],
             doc: "Close the current view.",
             fun: quit,
             completer: None,
         },
         TypableCommand {
             name: "quit!",
-            alias: Some("q!"),
+            aliases: &["q!"],
             doc: "Close the current view.",
             fun: force_quit,
             completer: None,
         },
         TypableCommand {
             name: "open",
-            alias: Some("o"),
+            aliases: &["o"],
             doc: "Open a file from disk into the current view.",
             fun: open,
             completer: Some(completers::filename),
         },
         TypableCommand {
             name: "write",
-            alias: Some("w"),
+            aliases: &["w"],
             doc: "Write changes to disk. Accepts an optional path (:write some/path.txt)",
             fun: write,
             completer: Some(completers::filename),
         },
         TypableCommand {
             name: "new",
-            alias: Some("n"),
+            aliases: &["n"],
             doc: "Create a new scratch buffer.",
             fun: new_file,
             completer: Some(completers::filename),
         },
         TypableCommand {
             name: "format",
-            alias: Some("fmt"),
+            aliases: &["fmt"],
             doc: "Format the file using a formatter.",
             fun: format,
             completer: None,
         },
         TypableCommand {
             name: "indent-style",
-            alias: None,
+            aliases: &[],
             doc: "Set the indentation style for editing. ('t' for tabs or 1-8 for number of spaces.)",
             fun: set_indent_style,
             completer: None,
         },
         TypableCommand {
             name: "line-ending",
-            alias: None,
+            aliases: &[],
             doc: "Set the document's default line ending. Options: crlf, lf, cr, ff, nel.",
             fun: set_line_ending,
             completer: None,
         },
         TypableCommand {
             name: "earlier",
-            alias: Some("ear"),
+            aliases: &["ear"],
             doc: "Jump back to an earlier point in edit history. Accepts a number of steps or a time span.",
             fun: earlier,
             completer: None,
         },
         TypableCommand {
             name: "later",
-            alias: Some("lat"),
+            aliases: &["lat"],
             doc: "Jump to a later point in edit history. Accepts a number of steps or a time span.",
             fun: later,
             completer: None,
         },
         TypableCommand {
             name: "write-quit",
-            alias: Some("wq"),
+            aliases: &["wq", "x"],
```

Comment:
Is this also available in vim/kakoune? I was thinking :x could close the underlying buffer instead of the view

Reply:
I was going to tell you yes, but it looks like it's not [*strictly*](https://til.hashrocket.com/posts/2fdb6afb66-difference-between-wq-and-x) the same in vim.
`wq` always write the file to disk while `x` write it only if it has changed.
And in kakoune they are the same I think? I don't find anything talking about it and I don't know where to find it in the code.

Personally, I think these aliases are enough for most people.

Reply:
I can't seemed to find `:x` in kakoune.

Reply:
Oh yes you're right I forgot I had added it to my kakrc!
```
define-command x write-quit
```
There is even an issue about it:
https://github.com/mawww/kakoune/issues/3503